project/plugins.sbt

```diff
@@ -21,6 +21,9 @@ addSbtPlugin("org.mdedetrich" % "sbt-apache-sonatype" % "0.1.6")
 // allow access to snapshots for pekko-sbt-paradox
 resolvers += "Apache Nexus Snapshots".at("https://repository.apache.org/content/repositories/snapshots/")
 
+// See https://github.com/akka/akka-http/pull/3995 and https://github.com/akka/akka-http/pull/3995#issuecomment-1026978593
```

Comment:
This is necessary because we are getting many conflict `scala-xml` versions from various libraries. Note that this only applies to the sbt classpath for plugins and not pekko-grpc in general.